### DISCUSSION TITLE
Related Parts loss of information available to the User

### DISCUSSION THREAD

**Discussion by smithcv** [inventree/InvenTree#8635] [Q&A]
I'm using Inventree 0.16.9

This is what I have set for the PART_NAME_FORMAT

PART_NAME_FORMAT | {{ part.IPN if part.IPN }}{{ ' \| ' if part.name }}{{ part.name }}

Here is the difference between the old interface and the PUI.  I'm hoping there is a setting or something to get the concatenated Name vs the Name proper of the part.  Here is a side by side of the two and how all the revs look the same now because of this:
![InventreeSidebySide](https://github.com/user-attachments/assets/ba9a59ba-3764-462b-be46-50e34412e065)

**Comment by SergeoLacruz** [comment]
It looks the same here. But we use unique IPN for each part. So it is no problem here. In your case all parts have the same IPN identical to the description which is also the same for all parts.

**Comment by SchrodingersGat** [comment]
See new IPN column added in https://github.com/inventree/InvenTree/pull/8650
